pkg/asset/machines/openstack/machines.go

```diff
@@ -111,6 +111,10 @@ func provider(clusterID string, platform *openstack.Platform, mpool *openstack.M
 				Name: fmt.Sprintf("%s-%s", clusterID, role),
 			},
 		},
+		ServerMetadata: map[string]string{
+			"Name":               clusterID + "-worker",
```

Comment:
Probably better to do `fmt.Sprintf("%s-%s", clusterID, role)`

Reply:
I actually just did that in my local lol